db/migrate/20210920070835_add_time_stamp.rb

```diff
@@ -0,0 +1,8 @@
+class AddTimeStamp < ActiveRecord::Migration[6.1]
+  def change
+    change_column_default :search_results, :created_at, -> { 'CURRENT_TIMESTAMP' }
+    change_column_default :search_results, :updated_at, -> { 'CURRENT_TIMESTAMP' }
+    change_column_default :ad_results, :created_at, -> { 'CURRENT_TIMESTAMP' }
+    change_column_default :ad_results, :updated_at, -> { 'CURRENT_TIMESTAMP' }
```

Comment:
We normally don't add default value to `created_at`/`updated_at`. Rails already does this.

Reply:
https://github.com/rails/rails/issues/35493 getting this issue when using insert_all kub. maybe better way to fix it ?

Reply:
Then we should prepare the data up first.

```ruby
bulk_data = {
  ...
  created_at: Time.current,
  updated_at: Time.current
}
Foo.insert_all(bulk_data)

```